### PR TITLE
Strange Reagent Change

### PR DESCRIPTION
**Pull request by Shadeykins**
**What does this PR do:**
~~This is an alternative that maintains the ability to raise NO_SCAN species but makes SR something difficult to obtain (requires mining). I am admittedly not sold that NO_SCAN should be revived (and personally in favor of #10844 by and large), but there was a lot of people going on about that so I figured I should make an alternative.~~

_To make it clear, this was an idea I've had rolling around in my head for awhile (prior to any of the recent medical reworks). Outside of being initially posted as an alternative, the underlying logic/origins of this idea are detached from the other medical PRs. If you have a concern about the changes, please direct them to me in a comment below (don't bug the maints, bug Shadey). Thanks!_

Right now SR is more or less a throw-away drug which you get a boatload of at roundstart. It not only reduces the point of the hypospray (that being a general cure-all that the CMO can use at their discretion/in emergencies) into little more than a reagent dispenser for making SR.

I also feel as though this is a reasonable change as there's a reciprocal interaction between Botany/Chemistry. Chemistry constantly helps Botany (they are more or less reliant upon them) and now, in turn, Botany can help out Chemistry.

* Strange Reagent is now a reductive reaction (90 units of reagents results in 30 units of SR)
* Strange Reagent now requires earthsblood instead of omnizine.
* Strange Reagent now has a volume floor of 10.

What does this mean?

90 units of reagents (30 Holy Water, 30 Unstable Mutagen, 30 Earthsblood) will result in 3 SR pills. Use them wisely.

**Changelog:**
:cl:
tweak: SR now needs 10 units to revive, simple mobs only need 1.
tweak: SR now requires earthsblood instead of omnizine.
tweak: SR is now a 1:2 reductive reaction.
/:cl:

### CODE REVIEW

**Comment by Necaladun** [comment]
How many sheets per pill is that?

**Comment by Shadeykins** [comment]
> 
> 
> How many sheets per pill is that?

1.5. 1 uranium sheet renders 20 uranium last I checked.

**Comment by Mitchs98** [comment]
An acceptable alternative to outright removing the ability to revive the races. Makes it more difficult to acquire but not outright require. IF either PR has to pass, I'd be more in favor of this one. 

Personally think just raising the amount of SR needed to 5u would of been acceptable...but this is a healthy compromise.

**Comment by Shadeykins** [comment]
> 
> 
> An acceptable alternative to outright removing the ability to revive the races. Makes it more difficult to acquire but not outright require. IF either PR has to pass, I'd be more in favor of this one.
> 
> Personally think just raising the amount of SR needed to 5u would of been acceptable...but this is a healthy compromise.

5u means it's still possible to make 45 pills at roundstart.

**Comment by AffectedArc07** [comment]
Again, slap this with `[Controversial]` please, maints

**Comment by davidchan13** [comment]
Unless you're requiring ground up hive cores, you've just exchanged one reagent that can be acquired in large quantities for a few minutes of work for another that can be acquired in large quantities for a few minutes of work by someone else. 

Mutagen is not rare and every chemist with a wiki can produce thousands of u for botany, cryo and SR.

Holy water can be produced in the extremely complicated steps of having a chaplain slap a 1000u water tank and walking away.

Omnizine at least required a competent botanist or cooperation of science and mining to get femtos. Even the greenest miner staying on the eastern asteroid could get 20 sheets of uranium before the 30 minute mark.

TL;DR this pr won't make SR rare, and would infect make it easier to get earlier in the round.

**Comment by Shadeykins** [comment]
> 
> 
> Unless you're requiring ground up hive cores, you've just exchanged one reagent that can be acquired in large quantities for a few minutes of work for another that can be acquired in large quantities for a few minutes of work by someone else.
> 
> Mutagen is not rare and every chemist with a wiki can produce thousands of u for botany, cryo and SR.
> 
> Holy water can be produced in the extremely complicated steps of having a chaplain slap a 1000u water tank and walking away.
> 
> Omnizine at least required a competent botanist or cooperation of science and mining to get femtos. Even the greenest miner staying on the eastern asteroid could get 20 sheets of uranium before the 30 minute mark.
> 
> TL;DR this pr won't make SR rare, and would infect make it easier to get earlier in the round.

The issue is that the CMO spawns with 30u of Omnizine, thereby enabling the creation of 90 bills.

I don't see any elegant means of resolution without removing Omnizine from the hypo (and even then, you can just dialysis it). If we wanted to go the botany route, earthsblood is always an option.

**Comment by davidchan13** [comment]
Carpotoxin would be a better choice. Though I think the other PR would be better since SR is too strong and easy with no real downside. 

Using a magical and "rare" chemical that nobody understands how or why it works should have dire consquences for the victim if it even works. Not just brushing the blood off and running back off like nothing happened.

**Comment by Blackmage198** [comment]
I don't get why people call SR "Too strong and easy".
It's not. Unless you SR someone within 5 minutes (Use a defib, what's wrong with you), you need to deal with sceptic/necro limbs, which unless you're a veteran doctor, *will* require at least 10 minutes of surgery for the inexperienced
You're not brushing off anything. Hell, you might just drop dead right after you get SR'd, depending on how late you get revived.

**Comment by Breenland** [comment]
I’m all for the volume floor, that just means how much you need to make 1u of it, yea? If so, that seems fair.

**Comment by Shadeykins** [comment]
> 
> 
> I’m all for the volume floor, that just means how much you need to make 1u of it, yea? If so, that seems fair.

Volume floor is how much you need to use in order for it to have any effect.

**Comment by Shadeykins** [comment]
@davidchan13 I'm not a fan of Carpotoxin because RNG, so I've switched it to Earthsblood. This requires a bit of investment from botany (who interacts with chemistry frequently as is) and a healthy use of the reagent grinder. Earthsblood is also slightly more involved (requiring chem, then botany) than simply smacking a rock once.

**Comment by Mitchs98** [comment]
Should update the changelog to reflect that, then. Personally like the idea of Earthsblood better, as if worst comes to worst medbay could grow their own if absolutely needed and it'd still take more time and effort than just smacking a rock.

**Comment by Shadeykins** [comment]
> 
> 
> Should update the changelog to reflect that, then. Personally like the idea of Earthsblood better, as if worst comes to worst medbay could grow their own if absolutely needed and it'd still take more time and effort than just smacking a rock.

Good catch. I updated the other bits but missed that one. :p

**Comment by Dinarzad** [comment]
> I'm not a fan of Carpotoxin because RNG

Just thought I'd put this out there. If you hack the pet vendor, you can get baby space carp eggs. When Baby space carp are cut with a sharp object (Wirecutters will do in a pinch!) you get carp fillets that have carpotoxin in them. Unsure on how much they have per fillet/fish though.
So if RNG is the main concern there, I wouldn't worry too much about it.

**Comment by Shadeykins** [comment]
> 
> 
> > I'm not a fan of Carpotoxin because RNG
> 
> Just thought I'd put this out there. If you hack the pet vendor, you can get baby space carp eggs. When Baby space carp are cut with a sharp object (Wirecutters will do in a pinch!) you get carp fillets that have carpotoxin in them. Unsure on how much they have per fillet/fish though.
> So if RNG is the main concern there, I wouldn't worry too much about it. The knowledge is a little obscure (Nobody uses it to make rezadone) buuuut. I suppose that mightbe the point.

From food? The amounts are typically really low, like.. 2-5 units per food item.

**Comment by Dinarzad** [comment]
Well the small amounts might make it suitably 'hard to obtain' like you'd be able to easily farm carpotoxin via the baby space carp, but it would be a very small rate of accumulation.

I've never actually ground up the carp fillets before to test out PRECISELY how much they contain however.
My point more was that it's not a worry for RNG dependency as you can do it that way. Just would be slow going.  Which IMO would be fine, it's incentive to not die as SR is in very limited quantities, an you'll be out for a while until they get enough for a new dosage. Which means more necrosis an means a generally longer time before you're back in it, without going as far as perma-death. Might also give formaldehyde more use, to prevent a body from rotting until SR can be administered.

**Comment by Mitchs98** [comment]
Against it using carpotoxin. Earthsblood is fine as it is. That's simply way, way too slow to farm any worthwhile amounts and taking things way to far. Formaldehyde also does nothing to prevent the necrosis that SR causes when used, at all. It prevents body sprite from rotting and giving the decay messages, nothing more.

**Comment by Dinarzad** [comment]
> Formaldehyde also does nothing to prevent the necrosis that SR causes when used, at all. It prevents body sprite from rotting and giving the decay messages, nothing more.

That was the idea. I wasn't saying it DID impede necrosis from SR, I was saying it would give a reason to make Formaldehyde to keep a body from rotting away as carpotoxin is gathered to make more SR, in the event there's Slime/Vox deaths going on left, right and center in rapid succession for some reason. For when SR has run out an you got a dead birb on the floor.
Given you only need a single unit in a corpse to keep it from decaying, there's barely a reason to make formaldehyde, as the coroner spawns with 30 units in his locker. Basically enough to process 30 bodies before needing a top up, which usually isn't necessary.

Being slow to craft is the point though, Earthsblood can be had extremely fast or extremely slowly depending on how much RNGesus cares for you today.  Carpotoxin is static, it may be slow-ish, but it's incredibly reliable. Having bred Carps before, they don't take that long to hit maturity, I've ended up with 10 or more fillets in just a few minutes, so even if we assume it's 1 unit of toxin per fillet, that's 10 units right there. It's enough to make SR available on hand when needed, but not enough to leave the chem fridge with 90 units of the stuff.

Pair that with needing 5 to 10 units to proc a revival, and you've got SR being functionally the same without making NO_SCAN races in a state of perma-death, it adds disincentive for dying without it being a permanent punishment and it incentivizes the use of other chems (Like formaldehyde) in a smart manner if things really hit the fan and all the SR is used up but need to buy a corpse time, it rewards a doctor for knowing their chems and what they do, and this is something that can easily passively done inside the chem lab as a fish bowl can easily fit onto a table, compared to a plant tray/soil lump.  Hell it's something ASSISTANTS could actually do to help out if someone were keen to do so/ask them to do if things got particularly nasty for the crew.

Earthsblood would require co-dependancy on Botany, or Science if botany is missing (To make your own plant tray) to set up, Carpotoxin requires nothing you can't get from a public tool storage and a vendor, at worst it's a multitool from cargo. In a low pop situation that's something you can easily get access to in the event botany is non-operational an science is being science an fucking off or there's no one doing RnD.

Being a Slime or a Vox is to knowingly be hard to revive, SR should be something much less frequent an harder to make, especially in vast numbers. This way it's slow to make, but it's something you can do with incredible reliability without changing SR's basic function, it just means there may be gap periods or just a few pills at a time.  You could use it's rarity in other events to help make it desirable on top, like adding SR pills as a possible Sol Trader item.

**Comment by Mitchs98** [comment]
Still not really sold on it being carpotoxin. You could fit a botany tray in the chemistry tray if that's your own argument towards its benefit. Earthsblood would still fit the niche of it being decently annoying to acquire. It would still require work and effort without being incredibly annoying to get. Especially with it being 10u of SR needed to revive someone. 

There shouldn't be any need to debate anything regardless in my opinion. The two SR PR's simply shouldn't exist as they are, or rather. The first shouldn't and SR shouldn't be made insanely hard to get. Two races shouldn't be permadeath races when nothing else is. Their benefits aren't even good enough to warrant it. Especially slimes' whose main 'benefit' is hard countered by a large brain damage modifier.

This PR with using Earthsblood makes SR a more valuable resource(Which to some point I agree with, given you can get 90 pills at round start as it is now), without making it too painful to acquire. You still have to be smart with resource management, you still have to actively farm Earthsblood plants.

You could, also, just order a plant tray from cargo if science isn't able to provide one. So that's also not a really valid argument.

**Comment by Dinarzad** [comment]
> Two races shouldn't be permadeath races when nothing else is. Their benefits aren't even good enough to warrant it. Especially slimes' whose main 'benefit' is hard countered by a large brain damage modifier.

To my understanding, both Slimes an Vox would be given a look over if the SR changes went through, to off set just that very thing.
I know Neca at least proposed giving both races either a buff or removing/reducing the debuffs they have now, it's possible the brain damage modifier would be eliminated altogether. And I agree with him that if we're going this route it SHOULD be reduced now that 'No clone' is actually a pretty damned major disadvantage.

As for ordering a plant tray from cargo, 'S fair enough. I legit forgot that was even a thing you could order since it's barely used/ordered as is.

I think Carpotoxin vs Earthsblood is sort of a 'Six of one, half dozen of another' situation. They're both ... Not necessarily HARD to acquire just take some effort.
Earthsblood would take more set up time, having to either mutate Ambrosia twice or find an Ambrosia Deus packet in maintenance to mutate once, but have more chemical pay off in the product itself, more Earthsblood per harvest.
Carpotoxin requires minimal to no set up time, all tools are available round start to obtain it, but it's less chemical per carp (Though one carp gives multiple fillets).  It's sort of a 'pick your poison' choice at this rate, they're of equal difficulty to obtain, they just differ in timing. (Slow to start, immediate pay out vs. Immediate start up, slow pay off over time.)

As for the rarity of SR, I'm inclined to agree. My stance in the other PRs was before the Defib changes. And especially before the proposed pain crit rework. All those things together leave me inclined to just make SR harder to make rather then to overhaul it's function. At least for now. Overhauling/removing it would be too many big changes at one time, as a personal opinion, better to see how the other PR's already merged in go an then go from there, because as it stands all of us are already using no longer accurate data/experiences.

**Comment by Mitchs98** [comment]
I'm aware of what Neca said. But quite frankly there'd need to be quite a few buffs to even consider making the two races permadeath. That said. I'm of the persuasion that if the crit rework PR passes SR shouldn't change at all, as it'll be used up a lot more and needed more. Or if you still insist, most definitely using uranium or earthsblood and possibly making it a 3:2 reaction I.E 2 pills instead of 1 or 3 pills per mix.

We aren't a high RP server. We're a chaotic medium RP server. This needs to be taken into account when 'making death more impactful'.

**Comment by davidchan13** [comment]
Kind of against Earthsblood because it's something Botany will go for regardless because of how gaia makes their trays self sufficient (which begs the question as to why it was added in the first place but this PR isn't the time or place for that.) 

Carptoxin isn't something used outside of Rezadone, meaning it's not something normally on hand. It has 2 reliable sources (Baby Carps via fish and Koi Beans via Botany), 2 RNG/Less reliable sources (Gold Slimes and Carp Migration Events) and 2 dangerous/unreliable sources (E.X.P.E.R.I-MENTOR and Space tears)

Rather than making more use for a chemical that already heals everything except brain damage, why not using something that is _strange_ with questionable use and multiple sources?

**Comment by Mitchs98** [comment]
> Kind of against Earthsblood because it's something Botany will go for regardless because of how gaia makes their trays self sufficient (which begs the question as to why it was added in the first place but this PR isn't the time or place for that.)
> 
> Carptoxin isn't something used outside of Rezadone, meaning it's not something normally on hand. It has 2 reliable sources (Baby Carps via fish and Koi Beans via Botany), 2 RNG/Less reliable sources (Gold Slimes and Carp Migration Events) and 2 dangerous/unreliable sources (E.X.P.E.R.I-MENTOR and Space tears)
> 
> Rather than making more use for a chemical that already heals everything except brain damage, why not using something that is _strange_ with questionable use and multiple sources?

Just because something is rarely used or doesn't have much of a use is not a viable argument when it comes to something as important as SR. Not in my opinion, at-least.

I sincerely do not understand why people are in such favor of nerfing SR/Slimes/Vox in the first place, but, whatever.

**Comment by Ozewse** [comment]
Why not remove roundstart omnizine? The CMO doesn't need an omni hypo and honestly, I've never seen a competent CMO do anything but throw their hypo at the chemist at the start of the round. Then work would have to be done to make omnizine through the existing channels. By all means, reduce the yield so you need a full bucket to make one pill, too.
[ I just don't think we need to put chemists through any more learning. ](https://cdn.discordapp.com/attachments/145035297246150656/547194055352647690/Capture.PNG)

**Comment by Mitchs98** [comment]
I agree with removing the omni from the hypo. I don't agree with something as outrageous as needing a full bucket for one pill. That's just..no.

**Comment by Ozewse** [comment]
Well, that was an exaggeration but given the recent PRs, I should probably clarify that, yes, that was an exaggeration.

**Comment by Shadeykins** [comment]
Omnizine is actually extremely easy to acquire using dialysis. I had given thought to this as well, but it wouldn't really achieve much difference.

This PR will probably be closed pending the crit rework. As Fox noted in his PR, there's no sense in looking at SR if the medical game is changed entirely.

**Comment by Jountax** [comment]
Carpotoxin is even easier to get from botany than earthblood is. The reason you never do is because bost botanists (and apparently most poeple) don't know that koibeans exist or that they contain it. I used to set up a little mini botany in science just to make rezadone to deliver to medbay because no one else ever did.

**Comment by Shadeykins** [comment]
I've updated the title and description to reflect that this is no longer an 'alternative' PR. I do feel it would be invalidated by the Crit Rework (should it be merged), but in lieu of that I think it's a good step to shakeup SR being more or less a throwaway no-think spam-out-to-every-dead-body-not-damaged chem.

The scarcity/resources required will give Chem/Botany another thing to do and hopefully shift play around it to treat it as more of an expensive chemical intended for patients who actually need it.

**Comment by variableundefined** [comment]
Regardless of whether the crit rework go in or not, it is time we let the dust settle over this making medbay more impactful business.
